SPECS/iptables/iptables.conf

```
@@ -0,0 +1,11 @@
+ip_tables
+iptable_filter
+iptable_mangle
+iptable_nat
+iptable_security
+ip6_tables
+ip6table_filter
+ip6table_mangle
+ip6table_nat
+ebt_ip
+nf_nat
```

```suggestion
nf_nat

```
EOF should have new line in Linux conf files, otherwise some .conf files will not be read correctly. (Likely not the case here since tests pass, but important to still address) Also please configure your editor to always include newline at EOF